components/kubearchive/base/migration-job.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: kubearchive-schema-migration-
+  namespace: kubearchive
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+spec:
+  parallelism: 1
+  backoffLimit: 4
+  template:
+    spec:
+      containers:
+        - name: migration
+          image: quay.io/fedora/python-311:latest
```

Comment:
I'm not sure about to use the latest tag here.

Reply:
Changed to a tag.

Reply:
Thank you Hector!